modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx

```diff
@@ -109,11 +109,33 @@ const CustomView: React.FC<React.HTMLAttributes<HTMLElement>> = () => {
 		makeFetch();
 	}, [objectViewId, dispatch]);
 
+	const removeLabelFromObjectView = (objectView: TObjectView) => {
+		const {objectViewColumns} = objectView;
+
+		const newObjectViewColumns: any = [];
+
+		objectViewColumns.map((viewColumn) => {
+			newObjectViewColumns.push({
+				objectFieldName: viewColumn.objectFieldName,
+				priority: viewColumn.priority,
+			});
+		});
```

Comment:
@danvb112 in this case, we can return the new object, like this:

```
const newObjectViewColumns: any = objectViewColumns.map(viewColumn => ({
        objectFieldName: viewColumn.objectFieldName,
        priority: viewColumn.priority,
    })
);
```

Reply:
Sure, Thanks! :smile: